scripts/pylib/twister/twisterlib/platform.py

```diff
@@ -95,26 +94,16 @@ def load(self, board, target, aliases, data):
         self.name = target
         self.aliases = aliases
 
-        # Get data for various targets and use the main board data as a
```

Comment:
why remove this? and all changes in this file seems to ignore the variants: data.